### ISSUE TITLE
Pokemon Black 2 and White 2 save editing problem.

### ISSUE THREAD

**Issue by NinFanBoyFTW** [kwsch/PKHeX#1395]
Hi. I have had this problem for a while and sometimes it happens after I edit these saves in pkhex and sometimes it doesn't. Basically what happens is that after I edit a save. Whenever I load the save I walk a few steps and all the sudden the prompt for the repel wearing off comes up. It is a minor glitch but sometimes when this happens in pokemon white 2, it completely freezes up my save not allowing me to even play the game. for some reason I am able to free up the save by using it in desmume and pressing random buttons or sometimes by fast forwarding, but still this is a pretty major glitch for pokemon white 2 at least. Also I know this isnt just me because I tried as a test to make a new save and edited that brand new save with pkhex and the same result happened. Usually I can reproduce this effect by either importing many boxes of pokemon into these saves or dragging pokemon from a random save directly into pokemon black 2 or white 2's PC boxes. I hope you can fix this because it could be very bad for people who dont use emulators.

**Comment by kwsch** [comment]
seeing how repel info is immediately after the pokedex, it could be possible that there's some incorrect bit writing happening.

```
new BlockInfo(0x21600, 0x04D4, 0x21AD6, 0x23F6E), // Pokedex
new BlockInfo(0x21B00, 0x0034, 0x21B36, 0x23F70), // Swarm and other overworld info - 2C - swarm, 2D - repel steps, 2E repel type
```

**Comment by NinFanBoyFTW** [comment]
Ok I did some testing and I was able to get the glitch to occur the first time trying it which kind of shows how it isnt very obscure.
[Pokemon White 2.zip](https://github.com/kwsch/PKHeX/files/1227139/Pokemon.White.2.zip)

All I did to make this happen was I loaded pokemon from a folder on my computer (my gen 5 master pokemon list) this contains about 700 pokemon in it and when I saved it with no other changes and went back into the game. I took a few steps around and then the game freezes for a bit. It then comes back with the repel prompt. On console I cant get it to unfreeze but on desmume I can for some reason but besides that another strange thing is that the major problem where it completely freezes up only seems to happen on white 2. Whenever this happens on Black 2 it only brings up the repel prompt.
